data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
> we should keep as much in Lua as posible.

i don't think so, imo we should increase lua coverage over server default behaviour, not move server to lua, it would be a disaster, we really shouldn't
> Having some codes in Lua can be faster than having them in c++.

even once in a blue moon cases, isn't a matter of server performance
> And gl using a 1.1 datapack with 1.3 or even 1.2 without doing changes.

that's why wiki provide server version for lua implementation, you're not suppose to use datapack 1.1 on 1.2, but any 1.2 datapack should be fully compatible with 1.2 server